### DISCUSSION TITLE
\textbf in hline text specification doesn't work

### DISCUSSION THREAD

**Discussion by yatescr** [lvjr/tabularray#538] [General]
Why doesn't this work? 

\begin{tblr}{
  colspec={lll},
  vlines,
  vline{2}={text={\textbf{=}}},
}
0 & pi & 3.14 \\
1 & pi & 3.14 \\
2 & pi & 3.14 \\
\end{tblr}

**Comment by lvjr** [comment]
It is an expansion bug. You can add a `\noexpand`.
```latex
\documentclass{article}
\usepackage{tabularray}
\begin{document}
\begin{tblr}{
colspec={lll},
vlines,
vline{2}={text={\noexpand\textbf{=}}},
}
0 & pi & 3.14 \\
1 & pi & 3.14 \\
2 & pi & 3.14 \\
\end{tblr}
\end{document}
```

PS: you will know how to enclose latex code in  questions in https://github.com/lvjr/tabularray/discussions/120.

**Comment by muzimuzhi** [comment]
Duplicate of #303.

**Comment by yatescr** [comment]
Thank you!
